gatewayconfig/bluetooth/characteristics/wifi_connect_characteristic.py

```diff
@@ -22,17 +24,26 @@ def __init__(self, service):
                 ["read", "write", "notify"], service)
         self.add_descriptor(WifiConnectDescriptor(self))
         self.add_descriptor(OpaqueStructureDescriptor(self))
-        self.wifi_status = ""
+        self.wifi_service = ""
```

Comment:
What do you think about using `None` instead?

Reply:
`self.wifi_service` and `self.wifi_password` should be guaranteed a String type. Is it more clear to initialize it with an empty String?